waterloop-site/src/components/PastGeeseTimeline/hooks/geese-images.ts

```diff
@@ -17,11 +17,12 @@ interface GooseImage {
   desc: string;
   cycleRight: React.EffectCallback;
   cycleLeft: React.EffectCallback;
+  selectGoose: any;
```

Comment:
This should not be the 'any' type, It should be type React.EffectCallback

Reply:
Changed, but there are a few complications that I'll show in my comment.

Reply:
hmmm, try this type: (index: number) => () => void;

Reply:
Honestly, did not know you could just do that with typescript. Thanks, it works now.

Reply:
sweet, no worries, Typescript is a bit of a learning curve at the start